### PR TITLE
KAA-937 Generate keys in build directory

### PR DESCRIPTION
**Pull request by Dubland**
/cc @forGGe @rasendubi

### CODE REVIEW

File: client/client-multi/client-c/listfiles/platform/cc32xx/CMakeLists.txt [hunk 1]
```diff
@@ -70,7 +70,8 @@ set(KAA_THIRDPARTY_LIBRARIES driver simplelink_nonos)
 
 set(KAA_BUILD_STATIC_ONLY 1)
 
-set(CC32XX_RSA_KEY ${KAA_SDK_DIR}/src/kaa/platform-impl/cc32xx/cc32xx_rsa_key.h)
+set(CC32XX_RSA_KEY ${CMAKE_CURRENT_BINARY_DIR}/cc32xx_rsa_key.h)
 
 execute_process(COMMAND rm ${CC32XX_RSA_KEY})
 execute_process(COMMAND java -jar ${KAA_SDK_DIR}/tools/pub_key_generator.jar ${CC32XX_RSA_KEY})
+include_directories (${CMAKE_CURRENT_BINARY_DIR})
```
Comment (rasendubi) [inline client/client-multi/client-c/listfiles/platform/cc32xx/CMakeLists.txt]: Remove space before `(`.

File: client/client-multi/client-c/listfiles/platform/esp8266/CMakeLists.txt [hunk 2]
```diff
@@ -48,4 +48,5 @@ set(KAA_INCLUDE_PATHS ${KAA_INCLUDE_PATHS} ${ESP8266_INCDIRS})
 set(KAA_BUILD_STATIC_ONLY 1)
 
 execute_process(COMMAND java -jar ${CMAKE_CURRENT_SOURCE_DIR}/tools/pub_key_generator.jar
-                ${CMAKE_CURRENT_SOURCE_DIR}/src/kaa/platform-impl/esp8266/rsa.h)
+                ${CMAKE_CURRENT_BINARY_DIR}/rsa.h)
+include_directories (${CMAKE_CURRENT_BINARY_DIR})
```
Comment (rasendubi) [inline client/client-multi/client-c/listfiles/platform/esp8266/CMakeLists.txt]: Same here

**Comment by rasendubi** [comment]
Rename commit to `KAA-937 Generate keys in build directory`

**Comment by rasendubi** [comment]
LGTM

**Comment by forGGe** [comment]
8ab4c66f6 lacks ticket number in description

**Pull request merged**